coral2.rst

```diff
@@ -0,0 +1,115 @@
+.. _coral2:
+
+==============
+CORAL2 Systems
+==============
+
+The LLNL and ORNL `CORAL2 systems <https://procurement.ornl.gov/rfp/CORAL2/>`_
+Lassen, Sierra, and Summit are pre-exascale supercomputers built by IBM.  They
+run a specialized software stack that requires additional components to
+integrate properly with Flux.  These components are provided as `Lmod
+<https://lmod.readthedocs.io/en/latest/>`_ modules on all three systems.
+
+To setup your environment to use these modules on the LLNL systems Lassen and
+Sierra, run:
+
+.. code-block:: sh
+
+  module use /usr/global/tools/flux/blueos_3_ppc64le_ib/modulefiles
+
+If you are using the ORNL system Summit, run:
+
+.. code-block:: sh
+
+  module use /sw/summit/modulefiles/ums/gen007flux/Core
+
+------------------
+Launching Flux
+------------------
+
+Launching Flux on CORAL2 systems requires a shim layer to provide `PMI
+<https://www.mcs.anl.gov/papers/P1760.pdf>`_ on top of the PMIx interface
+provided by the CORAL2 system launcher `jsrun`.  PMI is a common interface
+for bootstrapping parallel applications like MPI, SHMEM, and Flux.  To load this
+module, run:
+
+.. code-block:: sh
+
+  module load pmi-shim
+
+We also suggest that you launch Flux using jsrun with the following arguments:
```

Comment:
Not sure if you meant to backtick all mentions of `jsrun`. I actually think it looks fine without, but just wondered for consistency.